nu_stats/simulationold.py

```diff
@@ -0,0 +1,270 @@
+import numpy as np
```

Comment:
You can delete this file unless you are using it somewhere. It will live on in the repository's anyway.